corehq/apps/data_cleaning/tests/test_models.py

```diff
@@ -0,0 +1,103 @@
+from django.test import TestCase
+
+from casexml.apps.case.mock import CaseFactory
+from corehq.apps.data_cleaning.models import (
+    BulkEditChange,
+    EditActionType,
+)
+from corehq.form_processor.tests.utils import FormProcessorTestUtils
+
+
+class BulkEditChangeTest(TestCase):
+    domain = 'planet'
+    case_type = 'island'
+
+    def setUp(self):
+        factory = CaseFactory(domain=self.domain)
+        self.case = factory.create_case(
+            case_type=self.case_type,
+            owner_id='q123',
+            case_name='Vieques',
+            update={
+                'nearest_ocean': 'atlantic',
+                'town': 'Isabel Segunda',
+                'favorite_beach': 'Playa Bastimento',
+                'art': '  :)     ',
```

Comment:
there are many types of whitespaces. we should be sure to include `\t` and `\n`, as those are often the problematic ones (esp `\n`)

Reply:
0e52c6e282e